docs/resources/action_connection.md

```diff
@@ -146,36 +146,36 @@ Optional:
 
 Optional:
 
-- `content` (String) Serialized body content
-- `content_type` (String) Content type of the body
+- `content` (String) Serialized body content string length must be at least 1.
+- `content_type` (String) Content type of the body string length must be at least 1.
 
 
 <a id="nestedblock--http--token_auth--header"></a>
 ### Nested Schema for `http.token_auth.header`
 
 Optional:
 
-- `name` (String) Header name
-- `value` (String)
+- `name` (String) Header name string length must be at least 1.
+- `value` (String) string length must be at least 1.
 
 
 <a id="nestedblock--http--token_auth--token"></a>
 ### Nested Schema for `http.token_auth.token`
 
 Optional:
 
-- `name` (String) Token name
-- `type` (String) Token type
-- `value` (String, Sensitive) Token value
+- `name` (String) Token name string length must be at least 1.
+- `type` (String) Token type valid values are `SECRET`.
+- `value` (String, Sensitive) Token value string length must be at least 1.
```

Comment:
Given the inconsistencies in using `.` after the `currentDesc` and then not knowing if we need upper or lower case for the validation string, what about adding a small utility function to always append `.` at the end of the client-provided description and always upper casing the validation message?

Something like:

```go
func appendDotIfMissing(s string) string {
	if !strings.HasSuffix(s, ".") {
		return s + "."
	}
	return s
}
```